packages/amplify-category-geo/src/commands/geo.ts

```diff
@@ -0,0 +1,43 @@
+import { category } from '../constants';
+
+module.exports = {
+  name: category,
+  run: async (context: any) => {
```

Comment:
type should be `$TSContext` from `amplify-cli-core`

(this also applies anywhere else context is a function parameter)